Makefile

```diff
@@ -82,105 +147,57 @@ $(VECTORS_DIR)/%: $(SPECTEST_ROOTDIR)/%_${SPECTEST_VERSION}.tar.gz
 	-rm $(SPECTEST_ROOTDIR)/*.tar.gz
 
-clean-tests:
-	-rm -r test/generated
-
+#📝 gen-spec: @ Generate the spec tests.
 gen-spec: $(SPECTEST_GENERATED_ROOTDIR)
 
-##### TARGETS #####
-
-clean:
-	-rm $(GO_ARCHIVES) $(GO_HEADERS) $(OUTPUT_DIR)/*
-
-# Compile C and Go artifacts.
-compile-native: $(OUTPUT_DIR)/libp2p_nif.so $(OUTPUT_DIR)/libp2p_port
 
-compile-all: compile-native $(PROTOBUF_EX_FILES)
-	mix compile
-
-
-# Start application with Beacon API.
-start: compile-all
-	iex -S mix phx.server
-
-grafana-up:
-	cd metrics/ && docker-compose up -d
-
-grafana-down:
-	cd metrics/ && docker-compose down
-
-grafana-clean:
-	cd metrics/ && docker-compose down -v
-
-# Run an interactive terminal with the main supervisor setup.
-iex: compile-all
-	iex -S mix
-
-# Run an interactive terminal using checkpoint sync.
-checkpoint-sync: compile-all
-	iex -S mix run -- --checkpoint-sync https://sync-mainnet.beaconcha.in/
-
-# Install mix dependencies.
-
-deps:
-	sh scripts/install_protos.sh
-	$(MAKE) proto
-
-	cd native/libp2p_port; \
-	go get && go install
-	mix deps.get
+#🗑️ clean-tests: @ Remove the generated spec tests.
+clean-tests:
+	-rm -r test/generated
 
-# Run tests
-test: compile-all
-	mix test --no-start --exclude spectest
 
-# Run all spec tests
+#🔴 spec-test: @ Run all spec tests
 spec-test: compile-all $(PROTOBUF_EX_FILES) $(SPECTEST_GENERATED_ROOTDIR)
 	mix test --no-start test/generated/*/*/*
 
-# Run all spec tests for a specific config (e.g. mainnet)
+#🔴 spec-test-config-%: @ Run all spec tests for a specific config (e.g. mainnet)
 spec-test-config-%: compile-all $(PROTOBUF_EX_FILES) $(SPECTEST_GENERATED_ROOTDIR)
 	mix test --no-start test/generated/$*/*/*
 
-# Run all spec tests for a specific runner (e.g. epoch_processing)
+#🔴 spec-test-runner-%: @ Run all spec tests for a specific runner (e.g. epoch_processing)
 spec-test-runner-%: compile-all $(PROTOBUF_EX_FILES) $(SPECTEST_GENERATED_ROOTDIR)
 	mix test --no-start test/generated/*/*/$*.exs
 
-# Run spec tests for mainnet config, for the specified runner.
+#🔴 spec-test-mainnet-%: @ Run spec tests for mainnet config, for the specified runner.
 spec-test-mainnet-%: compile-all $(PROTOBUF_EX_FILES) $(SPECTEST_GENERATED_ROOTDIR)
 	mix test --no-start test/generated/mainnet/*/$*.exs
 
-# Run spec tests for minimal config, for the specified runner.
+#🔴 spec-test-minimal-%: @ Run spec tests for minimal config, for the specified runner.
 spec-test-minimal-%: compile-all $(PROTOBUF_EX_FILES) $(SPECTEST_GENERATED_ROOTDIR)
 	mix test --no-start test/generated/minimal/*/$*.exs
 
-# Run spec tests for general config, for the specified runner.
+#🔴 spec-test-general-%: @ Run spec tests for general config, for the specified runner.
 spec-test-general-%: compile-all $(PROTOBUF_EX_FILES) $(SPECTEST_GENERATED_ROOTDIR)
 	mix test --no-start test/generated/general/*/$*.exs
 
+#✅ lint: @ Check formatting and linting.
 lint:
 	mix format --check-formatted
 	mix credo --strict
 
+#✅ fmt: @ Format Go code.
```

Comment:
This formats _all_ code, including Rust and Elixir.

Reply:
Good catch!